ci/ruby.yml

```diff
@@ -7,18 +10,14 @@ on:
     branches: [ master ]
 
 jobs:
-  build:
+  test:
 
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Ruby 2.6
```

Comment:
Can we keep the `name` for consistency?  `Set up Ruby` is fine, we needn't have the version number.

Reply:
OK, I wanted to avoid hardcoding the version twice, that works too.